### ISSUE TITLE
Trouble with applying marathon gradle plugin

### ISSUE THREAD

**Issue by sundetova** [MarathonLabs/marathon#460]
Hello, I tried to use marathon gradle plugin, but receive next error: 

```
Could not find ch.frankel.log4k:slf4k-api:1.0.0.
Searched in the following locations:
  - https://dl.bintray.com/qameta/maven/ch/frankel/log4k/slf4k-api/1.0.0/slf4k-api-1.0.0.pom
  - https://dl.google.com/dl/android/maven2/ch/frankel/log4k/slf4k-api/1.0.0/slf4k-api-1.0.0.pom
  - https://repo.maven.apache.org/maven2/ch/frankel/log4k/slf4k-api/1.0.0/slf4k-api-1.0.0.pom
Required by:
    project :appName > com.malinskiy.marathon:marathon-gradle-plugin:0.6.1 > com.malinskiy.marathon:core:0.6.1
```

My app build.gradle:
```
plugins {
    id 'com.android.application'
    id 'kotlin-android'
    id 'kotlin-android-extensions'
    id 'marathon' version '0.6.1'
}
```

settings.gradle:
```
pluginManagement {
    repositories {
        maven { url = uri("https://dl.bintray.com/qameta/maven") }
        google()
        mavenCentral()
    }
    resolutionStrategy {
        eachPlugin {
            if (requested.id.id == "marathon") {
                useModule("com.malinskiy.marathon:marathon-gradle-plugin:0.6.1")
            }
        }
    }
}

rootProject.buildFileName = "build.gradle"
include ':appName'
```
I tried to add to my app build.gradle:
`implementation 'ch.frankel.log4k:slf4k-api:1.0.0'
`
But no success. Could you help me?

**Comment by Malinskiy** [comment]
Hey @sundetova! This library is coming from the jCenter. We have not done (as of yet) any review of the dependencies coming from jCenter in regards to their shutdown beginning of the next year.

For now, can you please add the jCenter to the repositories list? If that is not possible, your only option is to switch to CLI as recommended by the docs in the first place.

**Comment by sundetova** [comment]
Thank you for the answer. I added it to settings.gradle repository:
```
pluginManagement {
    repositories {
        mavenCentral()
        maven { url = uri("https://dl.bintray.com/qameta/maven") }
        google()
        jcenter()
    }
    resolutionStrategy {
        ...
}
```
but received next error:
```
Unable to find method 'com.fasterxml.jackson.core.JsonFactory.requiresPropertyOrdering()Z'.
Possible causes for this unexpected error include:
Gradle's dependency cache may be corrupt (this sometimes occurs after a network connection timeout.)
Re-download dependencies and sync project (requires network)

The state of a Gradle build process (daemon) may be corrupt. Stopping all Gradle daemons may solve this problem.
Stop Gradle build processes (requires restart)

Your project may be using a third-party plugin which is not compatible with the other plugins in the project or the version of Gradle requested by the project.

In the case of corrupt Gradle processes, you can also try closing the IDE and then killing all Java processes.
```
I guess that adding jcenter cause some incompatibility problem?

**Comment by sundetova** [comment]
By the way, I run marathon with cli, first time was successfull. But then I receive next error, also same situation with emulators:I can run marathon only first time:
```
1849752 bytes in 3.3590)
D 14:57:01.391 [main @coroutine#301] <AndroidDevice> /data/local/tmp/market-develop-debug.apk synced in 693ms
E 14:57:21.396 [main @coroutine#301] <AndroidAppInstaller> Error while installing kz.market.dev, /Users/**/projectFolder/apps/market/app/build/outputs/apk/develop/debug/market-develop-debug.apk on FA6AV0301411
com.malinskiy.marathon.android.exception.InstallException: Timeout transferring /Users/**/projectFolder/apps/market/app/build/outputs/apk/develop/debug/market-develop-debug.apk
        at com.malinskiy.marathon.android.adam.AdamAndroidDevice.installPackage(AdamAndroidDevice.kt:272)
        at com.malinskiy.marathon.android.adam.AdamAndroidDevice$installPackage$1.invokeSuspend(AdamAndroidDevice.kt)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:32)
        at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:113)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:172)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:84)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at com.malinskiy.marathon.Marathon.run(Marathon.kt:64)
        at com.malinskiy.marathon.cli.ApplicationViewKt$main$1.invoke(ApplicationView.kt:51)
        at com.malinskiy.marathon.cli.ApplicationViewKt$main$1.invoke(ApplicationView.kt)
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody(SystemExitException.kt:74)
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody$default(SystemExitException.kt:72)
        at com.malinskiy.marathon.cli.ApplicationViewKt.main(ApplicationView.kt:29)
```
I checked, there are no installed app on device.
My marathonFile:
```
name: "sample-app tests"
outputDir: "build/reports/marathon"
retryStrategy:
  type: "fixed-quota"
  totalAllowedRetryQuota: 100
  retryPerTestQuota: 1
vendorConfiguration:
  type: "Android"
  applicationApk: "apps/market/app/build/outputs/apk/develop/debug/market-develop-debug.apk"
  testApplicationApk: "apps/market/app/build/outputs/apk/androidTest/develop/debug/market-develop-debug-androidTest.apk"
  autoGrantPermission: true
  instrumentationArgs:
    debug: "false"
  applicationPmClear: false
  testApplicationPmClear: false
  vendor: ADAM
  waitForDevicesTimeoutMillis: 60000
  allureConfiguration:
    enabled: false
deviceInitializationTimeoutMillis: 180000
```

**Comment by Malinskiy** [comment]
Closed in 0.7.0 by #573
